etc/exceptions-exclusions.csv

```diff
@@ -0,0 +1,41 @@
+DocId,Namespace,Type,Member,Reason,linux,osx,win
```

Comment:
Great idea of using an exclusion file for manual auditing. I was thinking something more complicated :-)